crates/torii/sqlite/src/executor/mod.rs

```diff
@@ -448,13 +459,67 @@ impl<'c, P: Provider + Sync + Send + 'static> Executor<'c, P> {
                     )
                 })?;
                 let mut entity_updated = EntityUpdated::from_row(&row)?;
-                entity_updated.updated_model = Some(entity);
+                entity_updated.updated_model = Some(entity.ty.clone());
                 entity_updated.deleted = false;
 
                 if entity_updated.keys.is_empty() {
                     warn!(target: LOG_TARGET, "Entity has been updated without being set before. Keys are not known and non-updated values will be NULL.");
                 }
 
+                // Handle historical entities similar to historical event messages
+                let mut entity_counter: i64 = sqlx::query_scalar::<_, i64>(
```

Comment:
So everything will belong to the same `entities_historical` table?

Running this code on dojo started, both `Moves` and `Position` (which are respectively an event message and a model) will be in the same table.

Technically we agree, they are the same.

It's to clarify, and if it's the direction we take, do we want to remove the `event_messages_historical` table? We may also update the endpoint to query historical messages in GraphQL to actually support "historical entities".

Reply:
Otherwise, it works fine for both `dojo::event` and `dojo::model`.

Reply:
Not sure I understand. Event messages are on their own table `event_messages_historical` and entities on `entities_historical`. Moves and Postion are both normal models, not events.

Reply:
Then the behavior may not be correct, running Torii on this PR, everything ends up in the same table `entities_historical` in my case.
Will double check my setup then.

Reply:
<img width="1592" alt="image" src="https://github.com/user-attachments/assets/544a88ef-4a64-412f-8142-aaedf8555c2f" />
vs
<img width="1554" alt="image" src="https://github.com/user-attachments/assets/11fd9603-93a1-4e07-866c-a62e504151f7" />

Reply:
Heu ok... `Moves` is actually not an event in this example, but it is in other. My bad.